src/meta/src/controller/catalog.rs

```diff
@@ -274,8 +278,30 @@ impl CatalogController {
             .into_tuple()
             .all(&txn)
             .await?;
+
+        let parallel_unit_to_worker = Self::get_parallel_unit_to_worker_map(&txn).await?;
```

Comment:
Not related to this PR, we might need to cache it as well when we dully deprecate parallel units and maintain the fragment mapping in memory.